fudy/NoteScreen/Component/FramelessWindow.qml

```diff
@@ -111,15 +114,50 @@ ApplicationWindow {
 		onDoubleClicked: function() {
 			root.doubleClicked();
 		}
+
+		onPositionChanged: function(mouse) {
+			if (mouse.x >= parent.width - internal.margin && mouse.y >= parent.height - internal.margin) {
+				mouseArea.cursorShape = Qt.SizeFDiagCursor;
+			} else {
+				mouseArea.cursorShape = Qt.ArrowCursor;
+			}
+		}
 	}
 
-	ColumnLayout {
-		anchors.fill: parent
-		spacing: 0
+	Flickable {
+		id: scrollView
```

Comment:
It is flickable componet and its name is scrollView?

Reply:
I changed the id.